p4-16/psa/PSA.mdk

```diff
@@ -1028,33 +1025,35 @@ with defined contents.  Its value is the one that was assigned to the
 clone was created.
 
 
-### Multicast copies
+### Multicast and clone copies
```

Comment:
All of the description in this section now applies equally for cloned packets as well as multicast-replicated packets, so update the section title, too.